src/Components/NavBar.jsx

```diff
@@ -12,11 +12,16 @@ const NavComponent = () => {
             <Nav.Link href="/">Home</Nav.Link>
             <Nav.Link href="/articles">Articles</Nav.Link>
             <NavDropdown title="Topics" id="basic-nav-dropdown">
-              <NavDropdown.Item href="/football">Football</NavDropdown.Item>
-              <NavDropdown.Item href="/coding">Coding</NavDropdown.Item>
-              <NavDropdown.Item href="cooking">Cooking</NavDropdown.Item>
+              <NavDropdown.Item href="/topics/football">
+                Football
+              </NavDropdown.Item>
+              <NavDropdown.Item href="/topics/coding">Coding</NavDropdown.Item>
+              <NavDropdown.Item href="/topics/cooking">
```

Comment:
You could improve this by having this list of topic dropdown informed by an API call to your GET topics API request, rather than hardcoding them here
This would mean if another topic were added to the site, you wouldn't need to go back and hardcode in the new category